doc/manual/calling-c-and-fortran-code.rst

```diff
@@ -86,7 +83,7 @@ of an environment variable, one makes a call like this::
     julia> bytestring(path)
     "/bin/bash"
 
-Note that the argument type tuple must be written as ``(Ptr{UInt8},)``,
```

Comment:
I take it back, since the `bytestring` method accepts a `Cstring`, it's not crazy to use a `Cstring` as a return value.  e.g. `bytestring(ccall(:strdup, Cstring, (Cstring,), "hello"))` works (although it has a memory leak).